frontend/src/components/HomePage/Profiles/listOfSkills.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from './profiles.module.css';
+
+function LOS ({ user }) {
```

Comment:
this works for rendering a list of user's skills ! i would rename it to `UserListOfSkills` just so it's more apparent than using an acronym